src/Sarif/GitHelper.cs

```diff
@@ -87,7 +87,7 @@ public string GetCurrentCommit(string repoPath)
         public string GetBlame(string filePath)
         {
             return GetSimpleGitCommandOutput(
-                Path.GetDirectoryName(filePath),
+                GetTopLevel(filePath),
```

Comment:
>GetTopLevel [](http://example.com/codeflow?start=16&length=11)

@AvremelM, here is your fix, thanks for providing it.

Reply:
@michaelcfanning  Thank you! But this is only half of it, please see: 
https://github.com/microsoft/sarif-sdk/pull/2684#issuecomment-1597431868
as well as my Review suggestion on this PR